contracts/eosio.bios/include/eosio.bios/eosio.bios.hpp

```diff
@@ -254,6 +294,23 @@ namespace eosiobios
 
       typedef eosio::multi_index<"abihash"_n, abi_hash> abi_hash_table;
 
+      struct [[eosio::table]] resource_config {
+         uint64_t key; // This will be the primary key
+         uint64_t ram_price;
```

Comment:
can you add units here as comments, this will make the table easier to understand